xaynetwork/xayn_discovery_app
# TB-4214 Fix experiments bug so it appears in Mixpanel

### What 🕵️ 🔍

Identity param in Mixpanel API doesn't accept "Set" type, but "List" type. So we changed the identity param for experiments to be List instead of Set.

example user: https://eu.mixpanel.com/project/2712349/view/3248657/app/profile#distinct_id=583d1928-c246-4ca2-9dc8-b526819fcf15

----------

### How to test (please adjust template) 🥼 🔬
- Open the app
- Check you're subscribed to experiments from the Feature screen
- Open Mixpanel dashboard
- Check your logged-in user properties 

----------

### Screenshots 📸 📱

<img width="280" src="https://user-images.githubusercontent.com/12827629/192472009-dedeb6fe-94f5-4af1-8065-b9c28602f68c.png">

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-4214)

----------
